commands/blackjack.js

```diff
@@ -0,0 +1,199 @@
+const { SlashCommandBuilder, ButtonBuilder, ActionRowBuilder, ButtonStyle } = require('@discordjs/builders');
+const _ = require('lodash');
+const Deck = require('card-deck');
+const { DECK } = require('../constants');
+
+let deckOf52Cards;
+
+const convertAceToOne = (cards, cardSum) => {
+  for (var i = 0; i < cards.length; i++) {
+    if (cards[i].includes('A')) {
+      cards[i] = cards[i].replace('A', '1');
+      cardSum = cardSum - 10;
+      break;
+    }
+  }
+
+  return [cards, cardSum];
+};
+
+const getCardValue = (card) => {
+  let cardValue;
+  switch (card.substring(card.length - 1)) {
+    case 'T':
+    case 'J':
```

Comment:
should we really be storing the deck in a data structure like this? We dont need to have exact mappings of hand to values in our data structure, We can just store J Q K as 11,12,13 and here we can just say if value > 10 then cardValue = 10